playwright/helpers.ts

```diff
@@ -38,7 +38,14 @@ export async function authenticate(): Promise<{ refresh: string; access: string
 
   const { access, refresh } = await resp.json() as JSONResponse;
 
-  return { access, refresh };
+  // Extract cookies from response headers for non-dev environments
+  const cookies: string[] = [];
```

Comment:
This is shadowing the `cookies` argument in the function, we could add the default value to the argument and remove this declaration
```suggestion
```